packages/amplify-opensearch-simulator/src/index.ts

```diff
@@ -347,3 +353,16 @@ export const getPathToOpenSearchBinary = async (pathToOpenSearchLocal?: string):
   }
   return join('opensearchLib', 'bin', 'opensearch');
 };
+
+export const getPackageAssetPaths: GetPackageAssetPaths = async () => [relativePathToOpensearchLocal];
+
+export const getOpensearchLocalDirectory = () => {
+  let opensearchLocalDir = join(pathManager.getAmplifyLibRoot(), packageName);
+  try {
+    opensearchLocalDir = pathManager.getAmplifyPackageLibDirPath(packageName);
```

Comment:
if `getAmplifyPackageLibDirPath` it should just fail. It means something else has gone wrong with the lib setup